travis-texbuild.sh

```diff
@@ -5,7 +5,7 @@ if [[ "$TRAVIS" != "true" ]]; then
   TRAVIS_BUILD_DIR=$SCRIPT_PATH
   TRAVIS_BUILD_NUMBER="0000"
 fi
-export CONFIG_FILE="$TRAVIS_BUILD_DIR/.travis/tex-config.ini"
+export CONFIG_FILE="$TRAVIS_BUILD_DIR/.travis.yml"
 # Get the tex-scheme config option
 export texscheme=$(awk -F "=" '/tex-scheme/ {print $2}' "$CONFIG_FILE")
```

Comment:
Now, this line is supposed to get the `tex-scheme` option from the `.ini`-file. However, if the options are put under `env: global:` in the `.travis.yml`-file then the variable `$texscheme` should be available (given that you rename `tex-scheme` in the `.travis.yml`-file to `texscheme`). I suggest using the `sh` names (`texscheme` and `pushtype`) instead of the ones I made in the file (`tex-scheme` and `push-type`)

Reply:
I suggest a that we now do a different way for reading the variables via some `yaml` interpreter, and renaming them in the `.travis.yml` to match the shell script.